README.md

```diff
@@ -27,13 +27,33 @@ compatibility. When possible, upgrade your Conjur version to match the
 when using integrations, choose the latest suite release that matches your Conjur version. For any 
 questions, please contact us on [Discourse](https://discuss.cyberarkcommons.org/c/conjur/5).
 
-### From PyPI
+## Supported Platforms
+
+- Red Hat Enterprise Linux (7, 8)
+- Windows
+- macOS
```

Comment:
is this currently true? we probably shouldn't put it in the README until we have this support.

also, do you mean this to give supported platforms _for the CLI_? I should think for the Python 3 client, all that matters is the Python version you are using - platform is irrelevant. this heading should be clearer.

Reply:
You are right I was mixing what we currently have with what we want to have... so I will remove and focus on what we currently offer